packages/react/dialog/src/Dialog.tsx

```diff
@@ -178,12 +178,17 @@ DialogContent.displayName = CONTENT_NAME;
 
 /* -----------------------------------------------------------------------------------------------*/
 
+type RemoveScrollProps = React.ComponentProps<typeof RemoveScroll>;
 type DialogContentTypeElement = DialogContentImplElement;
 interface DialogContentTypeProps
-  extends Omit<DialogContentImplProps, 'trapFocus' | 'disableOutsidePointerEvents'> {}
+  extends Omit<DialogContentImplProps, 'trapFocus' | 'disableOutsidePointerEvents'> {
+  allowPinchZoom?: RemoveScrollProps['allowPinchZoom'];
+  removeScrollbar?: RemoveScrollProps['removeScrollBar'];
```

Comment:
Thoughts on remapping these to different names? I wasn't sure if we were cool with these or not.